motor_names:
  - "DRIVE_LEFT_FRONT_MOTOR"
  - "DRIVE_LEFT_BACK_MOTOR"
  - "DRIVE_RIGHT_FRONT_MOTOR"
  - "DRIVE_RIGHT_BACK_MOTOR"
  - "DRIVE_BACK_LEFT_1_MOTOR"
  - "DRIVE_BACK_RIGHT_1_MOTOR"
  - "TURRET_MOTOR"
  - "IDK it doesnt matter lol"